doing-business-with-us/index.html

```diff
@@ -25,19 +25,20 @@
 
     <section class="block
                     block__bg
-                    block__border-top
-                    block__border-bottom
+                    block__border
+                    block__flush-sides
                     block__business-step_container
-                    block__flush-sides">
+                    ">
         <h1 class="h2">Opportunities to work with us</h1>
         <div class="content-l content-l__main">
-            <div class="block
-                        block__border-bottom
-                        block__business-step
+            <div class="content-l_col
+                        content-l_col-1-3
+                        block
                         block__padded-bottom
+                        block__border-bottom
                         block__flush-top
-                        content-l_col
```

Comment:
Hmm, I wondered if order mattered. Do you know if that's documented somewhere? I didn't see any change, but as we know manually checking local against refresh or beta isn't the easiest.